ComputerInterface/Views/GameSettings/ColorSettingView.cs

```diff
@@ -35,10 +35,10 @@ public override void OnShow(object[] args)
         private void Redraw()
         {
             var str = new StringBuilder();
-            str.BeginColor(_color).Repeat("=", SCREEN_WIDTH).EndColor().AppendLine();
+            str.BeginColor("ffffff").Repeat("=", SCREEN_WIDTH).EndColor().AppendLine();
```

Comment:
Assuming you made this change because `_color` was causing null exceptions? This suggestion would fix it if that's the case.
```suggestion
            str.BeginColor(_color ?? Color.white).Repeat("=", SCREEN_WIDTH).EndColor().AppendLine();
```

Reply:
not sure if i missed an update but the colour view i built with the source code and the colour view in the release looked different so i must've rushed to finish it or whatever